src/main/java/org/snomed/aag/rest/util/MetadataUtil.java

```diff
@@ -0,0 +1,63 @@
+package org.snomed.aag.rest.util;
+
+import org.ihtsdo.otf.rest.client.terminologyserver.pojo.Branch;
+import org.snomed.aag.data.Constants;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class MetadataUtil {
+	private MetadataUtil() {
+
+	}
```

Comment:
I've added this class so I wasn't hiding the extraction logic in a service method.